api/package-lock.json

```diff
@@ -16,21 +16,32 @@
         "express": "^5.0.1",
         "express-rate-limit": "^7.4.0",
         "google-auth-library": "^9.14.2",
+<<<<<<< HEAD
```

Comment:
Looks like there are conflicts in this file still.  They will need to be resolved.  I recommend just checking the lockfile out to the previous version and re-installing with merged dependencies to hopefully minimize the lockfile conflicts.